test/e2e/websocket-solana-mocks.ts

```diff
@@ -0,0 +1,90 @@
+// eslint-disable-next-line @typescript-eslint/no-shadow
+import { WebSocket } from 'ws';
+import LocalWebSocketServer from './websocket-server';
+
+/**
+ * WebSocket Solana mocks
+ * This function should be called after the WebSocket server is started
+ */
```

Comment:
:information_source:  these were the existing mocks, which were inside common solana, now have been moved to a separate file for making easier to work with